src/cli.rs

```diff
@@ -64,6 +64,19 @@ pub struct Ping_ {
     pub nix_file: NixFile,
 }
 
+///  Add parameter to define the shell file to use
+#[derive(StructOpt, Debug)]
+pub struct NixShellFile {
+    /// The .nix file to use
```

Comment:
Because we use `locate_file` when loading the file, the documentation should say that it must be a single file's name, and not be `../foo.nix` or `./bar/baz.nix` etc.

Reply:
Does that mean we won't be able to disambiguate between, say, a shell.nix file in a project subdirectory and a shell.nix file in a git submodule of that project?